source/operations/concepts/availability-and-resiliency.rst

```diff
@@ -0,0 +1,153 @@
+   This topology improves resiliency to the loss of a single node, or even a storage controller on that node.
+
+   .. figure:: /images/availability/availability-erasure-sharding-striped.svg
+      :figwidth: 100%
+      :align: center
+      :alt: Diagram of a 16 node x 8 drive per node cluster, consisting of 8 16-drive erasure sets striped evenly across each node.
+
+      In this 16 x 8 deployment, MinIO would calculate 8 erasure sets of 16 drives each.
+      It allocates one drive per node across the available nodes to fill each erasure set.
+      If there were 8 nodes, MinIO would need to select 2 drives per node for each erasure set.
+   
+   In the above topology, the pool has 8 erasure sets of 16 drives each striped across 16 nodes.
+   Each node would have one drive allocated per erasure set.
+   While losing one node would technically result in the loss of 8 drives, each erasure set would only lose one drive each.
+   This maintains quorum despite the node downtime.
+
+Each erasure set is independent of all others in it's same pool.
+   If one erasure set becomes completely degraded, MinIO can still perform read/write operations on other erasure sets.
+
+   .. figure:: /images/availability/availability-erasure-set-failure.svg
+      :figwidth: 100%
+      :align: center
+      :alt: Diagram of a MinIO multi-pool deployment with one failed erasure set in a pool
+
+      One pool has a degraded erasure set.
+      While MinIO can no longer serve read/write operations to that erasure set, it can continue to serve operations on healthy erasure sets in that pool.
+
+   Since erasure sets are independent, you cannot restore data to a completely degraded erasure set using other erasure sets.
+   You must use :ref:`Site <minio-site-replication-overview>` or :ref:`Bucket <minio-bucket-replication>` replication to create a :abbr:`BC/DR (Business Continuity / Disaster Recovery)`-ready remote deployment for restoring lost data.
+
+For multi-pool MinIO deployments, each pool requires at least one erasure set maintaining read/write quorum to continue performing operations.
+   If one pool loses all erasure sets, MinIO can no longer determine whether whether a given read/write operation would have routed to that pool.
+   MinIO therefore stops all I/O to the deployment, even if other pools remain operational.
+
+   .. figure:: /images/availability/availability-pool-failure.svg
+      :figwidth: 100%
+      :align: center
+      :alt: Diagram of a MinIO multi-pool deployment with one failed pool.
+
+      One pool in this deployment has completely failed.
+      MinIO can no longer determine which pool or erasure set to route I/O to.
+      Since continuing operations could result in consistency or collision if the degraded pool were to return to stable, MinIO halts all I/O on the deployment.
```

Comment:
resulting in consistency doesn't seem like it would be a problem?
Did you mean
> Since continuing operations could result in inconsistency or collision...

Reply:
Made a tweak